includes/admin/class-wc-admin-dashboard.php

```diff
@@ -246,6 +250,13 @@ private function status_widget_stock_rows() {
 
 			$transient_name   = 'wc_outofstock_count';
 			$outofstock_count = get_transient( $transient_name );
+			$lowstock_link    = 'admin.php?page=wc-reports&tab=stock&report=low_in_stock';
+			$outofstock_link  = 'admin.php?page=wc-reports&tab=stock&report=out_of_stock';
+
+			if ( false === $is_wc_admin_disabled ) {
+				$lowstock_link   = 'wp-admin/admin.php?page=wc-admin&type=lowstock&path=%2Fanalytics%2Fstock';
```

Comment:
The `wp-admin/` is not needed here as it's already prepended by `admin_url()` Same with the line below this.

Reply:
Thanks, that's what I get from copying it straight from the ticket. It's been updated.